mobile-widgets/qmlmanager.cpp

```diff
@@ -1583,6 +1583,18 @@ void QMLManager::setLibdcLog(bool value)
 	emit libdcLogChanged();
 }
 
+bool QMLManager::developer() const
+{
+	return m_developer;
+}
+
+void QMLManager::setDeveloper(bool value)
+{
+	m_developer = value;
+	DCDeviceData::instance()->setSaveLog(value);
+	emit developerChanged();
+}
+
 bool QMLManager::btEnabled() const
```

Comment:
I think you misunderstood my comment. I'm asking "why do we have the manager.developer interaction at all, why isn't this just a QML thing?" - we use the QMLmanager properties and functions for things that need to be shared between the QML and the C++ code. I don't see you use the m_developer variable anywhere in the C++ code, so I wonder why it's there... shouldn't you just make this a property in the QML code and act upon it?